docs/advanced.md

```diff
@@ -27,15 +27,15 @@ RKE2 will generate config.toml for containerd in `/var/lib/rancher/rke2/agent/et
 
 For advanced customization for this file you can create another file called `config.toml.tmpl` in the same directory and it will be used instead.
 
-The `config.toml.tmpl` will be treated as a Go template file, and the `config.Node` structure is being passed to the template. [This template](https://github.com/rancher/k3s/blob/master/pkg/agent/templates/templates.go#L16-L32) example on how to use the structure to customize the configuration file.
+The `config.toml.tmpl` will be treated as a Go template file, and the `config.Node` structure is being passed to the template. See [this template](https://github.com/rancher/k3s/blob/master/pkg/agent/templates/templates.go#L16-L32) for an example of how to use the structure to customize the configuration file.
 
 # Secrets Encryption Config
-RKE2 supports the secrets encryption at rest, and will do the following automatically:
+RKE2 supports encrypting Secrets at rest, and will do the following automatically:
```

Comment:
```suggestion
RKE2 supports encrypting secrets at rest, and will do the following automatically:
```

Reply:
A Secret is a specific type of Kubernetes resource, so it bears capitalizing.

Reply:
Can we include or something else to provide context for it's use in that manner?

Reply:
I could hyperlink it to https://kubernetes.io/docs/concepts/configuration/secret/

Reply:
That'd be awesome. Thanks!